modules/environment-roles/templates/terraform_restore_db_policy.json.tpl

```diff
@@ -0,0 +1,66 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": [
+        "iam:AttachRolePolicy",
+        "iam:CreatePolicy",
+        "iam:CreatePolicyVersion",
+        "iam:DeletePolicy",
+        "iam:DeletePolicyVersion",
+        "iam:DetachRolePolicy",
+        "iam:GetPolicy",
+        "iam:GetPolicyVersion",
+        "iam:GetRole",
+        "iam:ListAttachedRolePolicies",
+        "iam:ListPolicyVersions",
+        "rds:AddTagsToResource",
+        "rds:CreateDBInstance",
+        "rds:DeleteDBCluster",
+        "rds:DeleteDBInstance",
```

Comment:
Are `DeleteDBCluster` and `DeleteDBInstance` necessary? Is it so you can run `terraform destroy` once you're done? Just looks a little dangerous that it grants deletion permission on all databases!

Reply:
I know what you mean but I don't think we've got much choice. The existing databases have an ID of something like `db-postgres-instance-intg${random_id_from_terraform}` and the restored instance uses the same id. I tried to override it so I could be more specific in the script but it ignored me. They are there so we can run destroy on the cluster instances after we're done. There's nothing in the `tdr-scripts` repo that would allow us to delete the primary cluster though so I think it's safe enough.